tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Services/ProfileServiceTests.cs

```diff
@@ -426,6 +458,72 @@ public static async Task GetProfileDataAsync_GivenUserRepositoryDependency_FindB
                 r => r.GetByIdAsync(It.Is<string>(param => expectedUserId.Equals(param, StringComparison.Ordinal))));
         }
 
+        [Test]
+
```

Comment:
I think we can omit the `[Test]` attribute when we have one or more `[TestCase]` attributes.